RELEASE-NOTES.md

```diff
@@ -33,7 +33,7 @@
 
 1. Optimization for Sharing Parser with ANTLR Visitor improving the parsing performance of long SQL by 100%~1000%
 1. Use multiple threads to load metadata for different data sources
-1. Support `allow.range.query.with.inline.sharding` option
+1. Support `allow-range-query-with-inline-sharding` option
```

Comment:
previous release notes should be reserved.

Reply:
How should i deal witch it.

Reply:
All .md field keep previous code form?How about after 4.1.1. or rewrite doc as `Update to 4.*.* - 4.1.1`

Reply:
release-note is a history. just keep it original.

Reply:
I mean doc like ` _index.cn.md`
```
1. Update to 4.0.1 above.
2. Configure`allow-range-query-with-inline-sharding` to `true` (Default value is `false`).
3. A tip here: then each range query will be broadcast to every sharding table.
```